iceoryx_examples/automotive_soa/include/topic.hpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef IOX_EXAMPLES_AUTOMOTIVE_SOA_TOPIC_HPP
+#define IOX_EXAMPLES_AUTOMOTIVE_SOA_TOPIC_HPP
+
+#include "iceoryx_hoofs/cxx/serialization.hpp"
+#include "iceoryx_hoofs/cxx/type_traits.hpp"
+
+#include <chrono>
+#include <cstdint>
+
+constexpr uint32_t ONE_KILOBYTE = 1000U;
```

Comment:
One kilobyte should be `1024U`.

Reply:
You could also add `ONE_MEGABYTE = ONE_KILOBYTE * ONE_KILOBYTE`

Reply:
As 3Mb can be considered in the same order of magnitude as 4Mb compared to the other samples, I'll pursue this approach. Using `iceperf` is not an option as it does not use `popo::Listener` to wake-up and we don't want to measure a ping-pong.